config/tests/plugins/plugins_test.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package plugins
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	"k8s.io/test-infra/prow/plugins"
```

Comment:
This is what @BenTheElder was asking about in the SIG-Testing meeting. Specifically, once we move the Prow source to a new repo, these will become external imports that are pinned to a specific version. 

Can you discuss the potential for version skew problems and how we can deal with that if necessary? Will we need to bump our Prow go imports every time we bump Prow component versions?